### PR TITLE
Upgrade to .NET 6 and using trial versions of Infragistics WPF Nuget packages

### PR DESCRIPTION
**Pull request by vivekrathod**
I started learning through your YouTube video series on building the PrismOutlook application (thank you very much for the same). For me, it was convenient to use .NET 6, and also the versions of Infragistics Nuget packages there were paid versions which I did not have access to, so I used the trial versions. I am not sure if this may be of any use to others. In that case, please feel free to drop this PR.

### CODE REVIEW

**Comment by brianlagunas** [comment]
Hey, thanks for the PR. I don't mind accepting this PR, but please remove all references to the tool you used to generate this upgrade.

**Comment by vivekrathod** [comment]
I removed the references to the tool and the packages it added.

**Comment by vivekrathod** [comment]
Hi @brianlagunas, if you are busy and just haven't gotten to this yet, I understand. But if there is something I should change/correct, please let me know!

File: Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj [hunk 1]
```diff
@@ -1,100 +1,23 @@
...
-  <ItemGroup />
   <ItemGroup>
-    <Page Include="Views\ViewA.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
+    <PackageReference Include="Infragistics.WPF.DataTree.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.OutlookBar.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.Ribbon.Trial" Version="22.1.71" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```
Comment (brianlagunas) [inline Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj]: remove

File: Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj [hunk 2]
```diff
@@ -1,100 +1,23 @@
...
+    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
```
Comment (brianlagunas) [inline Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj]: remove

File: Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj [hunk 3]
```diff
@@ -1,100 +1,23 @@
...
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" />
```
Comment (brianlagunas) [inline Modules/PrismOutlook.Modules.Calendar/PrismOutlook.Modules.Calendar.csproj]: remove

File: PrismOutlook.Core/PrismOutlook.Core.csproj [hunk 4]
```diff
@@ -1,114 +1,19 @@
...
-      <Generator>SettingsSingleFileGenerator</Generator>
-      <LastGenOutput>Settings.Designer.cs</LastGenOutput>
-    </None>
+    <PackageReference Include="Infragistics.WPF.RichTextDocument.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.RichTextEditor.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.Undo.Trial" Version="22.1.71" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Prism.Wpf" Version="7.2.0.1367" />
+    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
```
Comment (brianlagunas) [inline PrismOutlook.Core/PrismOutlook.Core.csproj]: remove

File: PrismOutlook.Core/PrismOutlook.Core.csproj [hunk 5]
```diff
@@ -1,114 +1,19 @@
...
-    <None Include="packages.config" />
-    <None Include="Properties\Settings.settings">
-      <Generator>SettingsSingleFileGenerator</Generator>
-      <LastGenOutput>Settings.Designer.cs</LastGenOutput>
-    </None>
+    <PackageReference Include="Infragistics.WPF.RichTextDocument.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.RichTextEditor.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.Trial" Version="22.1.71" />
+    <PackageReference Include="Infragistics.WPF.Undo.Trial" Version="22.1.71" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```
Comment (brianlagunas) [inline PrismOutlook.Core/PrismOutlook.Core.csproj]: remove

File: PrismOutlook.Core/PrismOutlook.Core.csproj [hunk 6]
```diff
@@ -1,114 +1,19 @@
...
+    <PackageReference Include="Prism.Wpf" Version="7.2.0.1367" />
+    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" />
```
Comment (brianlagunas) [inline PrismOutlook.Core/PrismOutlook.Core.csproj]: remove

**Review by brianlagunas — CHANGES_REQUESTED** [review]
Please remove the unnecessary nuget packages

**Comment by vivekrathod** [comment]
Removed all these NuGet packages you pointed out. 
    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
    <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.0" />

**Review by brianlagunas — APPROVED** [review]

**Comment by brianlagunas** [comment]
Thanks for the PR. Sorry it took so long. I completely forgot about this.

**Pull request merged**